t/components-JetBackup.t

```diff
@@ -84,23 +84,12 @@ my $jb   = $cpev->get_blocker('JetBackup');
 
     my $jb_mock = Test::MockModule->new('Elevate::Components::JetBackup');
 
-    for my $os ( 'cent', 'cloud' ) {
+    for my $os ( 'cent', 'cloud', 'ubuntu' ) {
         set_os_to($os);
 
         my $expected_target_os = Elevate::OS::upgrade_to_pretty_name();
         is( $jb->_blocker_jetbackup_is_supported(), undef, "JetBackup is supported for upgrades to $expected_target_os" );
     }
```

Comment:
TRIVIAL: This whole block could go away now that Jetbackup is supported on all OSs

Reply:
this will come up again on 8->9....